data/hai/hai reveal 4 middle.txt

```diff
@@ -372,7 +372,7 @@ mission "Hai Reveal: Pirate Troubles [1]"
 				`	"I'd be glad to help. What can I do?"`
 					goto explain
 				`	"What issue?"`
-			`	"I understand that you just defended this system from human pirates. That is the issue that needs resolved.`
+			`	"I understand that you just defended this system from human pirates. That is the issue that needs resolving.`
```

Comment:
```suggestion
			`	"I understand that you just defended this system from human pirates. That is the issue that needs to be resolved.`
```

This isn't necessarily incorrect (especially as spoken dialogue), but this also isn't the simplest fix.

Suggesting the change to a more fitting fix, but dropping "to be" is such a very common convention in speech that I really think this one may be best left as is.

Reply:
I understand that "to be" is often dropped, but "needs resolved" is definitely not valid. "Needs to be resolved" would be fine.

Reply:
Spoken dialogue not being grammatically correct is wholly valid, as long as it is intended. ;)